e2e/playwright/editorFixture.ts

```diff
@@ -1,25 +1,27 @@
 import type { Page, Locator } from '@playwright/test'
 import { expect } from '@playwright/test'
+import { sansWhitespace } from './test-utils'
 
 interface EditorState {
   activeLines: Array<string>
   highlightedCode: string
   diagnostics: Array<string>
 }
 
-function removeWhitespace(str: string) {
-  return str.replace(/\s+/g, '').trim()
-}
 export class EditorFixture {
-  public readonly page: Page
+  public page: Page
 
-  private readonly diagnosticsTooltip: Locator
-  private readonly diagnosticsGutterIcon: Locator
-  private readonly codeContent: Locator
-  private readonly activeLine: Locator
+  private diagnosticsTooltip!: Locator
+  private diagnosticsGutterIcon!: Locator
+  private codeContent!: Locator
+  private activeLine!: Locator
 
   constructor(page: Page) {
     this.page = page
+    this.reConstruct(page)
+  }
+  reConstruct = (page: Page) => {
+    this.page = page
```

Comment:
This `reContsruct` method is added to all of the fixtures besides AuthenticatedApp and AuthenticatedTronApp, 

As this allows AuthenticatedTronApp.initialise to call this method to update the page property.